README.md

```diff
@@ -686,6 +686,27 @@ such behavior for `hidden` or `required`; use `accessor.hidden` and
 ``accessor.required` in your rendering code to determine whether a field wants
 to be hidden or required.
 
+## Warnings
```

Comment:
The section about validation errors should have a mention of the get error hook. You should add a section about the getError hook. You should mention that if there is an internally generated error message that trumps the one returned by the getError hook.